UI: Automatically populate 'Test Name' field after fetching test results

aayushdutt opened the pull request:
Signed-off-by: Aayush Dutt <aayushd100@gmail.com>

<!--
Thank you for contributing to Meshery! Before you submit this PR, please review 
the project's technical requirements and best practices, if you have not already:

- https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md

Quick list of contributing conventions:

1. Descriptive PR titles
  Include the component this PR addresses at the beginning of the title of the PR.
  e.g. `meshery:`, `mesheryctl:`, `ui:`

2. Build and test your changes before submitting a PR. 
  Please make sure you test your changes before you push them. Once pushed, a CI build
  will run across your changes and do some initial checks and linting. These checks run
  quickly. Please check the results for any fixes you may need to make. Reviewers/maintainers
  look for passing checks first before starting code review.

3. When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. 

For an overview of what the review process entails, please read our review guidelines:

- https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#contribution-conventions

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.

Thanks for contributing!
-->
#### Description
<!-- (description of the change included in this PR; why this change should be incorporated into the project). -->

This PR fixes #907     <!-- (put issue # here to ensure it is automatically closed upon merge of this PR) -->
![image](https://user-images.githubusercontent.com/25280541/80430670-ed2d4000-890c-11ea-909f-fdd5eed41e64.png)

#### Checklist
<!-- Place an '[x]' (no spaces) in all applicable fields. -->
- [x] [DCO](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) signed.
- [x] Title of the PR starts with area name, e.g. `meshery:`, `mesheryctl:`, `ui:`.

leecalcote commented:
✔️ Running a brief test, remaining on the /performance page awaiting the test to conclude, I see the randomly generated name now populated in the Test Name field. 

<img width="1231" alt="Screen Shot 2020-04-27 at 6 57 48 PM" src="https://user-images.githubusercontent.com/7570704/80432211-23d96b00-88b9-11ea-8467-8b37f3c8c464.png">

✖️ Running a brief test, navigating away from the /performance page before the test has concluded, the randomly generated name does not appear to be populated in the Test Name field.

![Untitled](https://user-images.githubusercontent.com/7570704/80432412-c8f44380-88b9-11ea-9916-f79c955aad90.gif)

aayushdutt commented:
@leecalcote  I have fixed the issue that you mentioned.
Also, now the test name is populated instantly after pressing 'Run Test' button (realised that the name is generated on the client-side).
Removed `testNameError` from the state as it's no longer required.

leecalcote commented:
@aayushdutt looking good. Works well.
